.travis.yml

```diff
@@ -1 +1,11 @@
 language: java
+
```

Comment:
These changes look unrelated to your fix, why did you need to add these changes to Travis?

Reply:
@DarthHater Travis is using `java11` by default for now and I'm forcing it to use `java8`